### DISCUSSION TITLE
FEATURE REQUEST: Load Rules from URL [Like Filters]

### DISCUSSION THREAD

**Discussion by madranet** [uBlockOrigin/uBlock-issues#2392] [Ideas] [answered]
Currently we can load Filter Lists from a URL, which makes it easy  to keep my filters synced across all my devices. I keep my filters in a `.txt` file on GoogleDocs, which I subscribe to from all my browsers and devices.

Unfortunately the same isn't possible with my UBO Rules. It would be great if there was the option to similarly 'subscribe' to a file at a  URL containing my Rules.

**Comment by ghost** [comment, marked as answer]
https://github.com/gorhill/uBlock/issues/1207#issuecomment-170608860

https://github.com/gorhill/uBlock/issues/1063

**Reply by madranet** [reply]
I don't see the relevance of either of those ???

**Reply by gothic-bum** [reply]
The relevance is that:

1. Gorhill has previously declined a request for subscribing to dynamic filtering rules, and he also stated that users will need to manually import them. 
2. It may be possible to translate dynamic filtering rules into static filters, which can then be added to a static filter list and subscribed to.
